### PR TITLE
Add update webhook scrpit and cleanup repos

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: scripts/update_webhook.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  abort("GITHUB_AUTH_TOKEN is needed to access the GitHub API")
+end
+
+repos = YAML.load_file('../config/repos.yaml').keys
+
+Octokit.configure do |c|
+  c.access_token = ENV['GITHUB_AUTH_TOKEN']
+end
+
+new_url = "http://prprocessor.theforeman.org/pull_request"
```
Comment (ekohl) [inline scripts/update_webhook.rb]: Can we use the HTTPS endpoint?

File: config/repos.yaml [hunk 2]
```diff
@@ -11,36 +11,45 @@ theforeman/chef-handler-foreman:
   redmine: chef
 theforeman/community-templates:
   redmine: foreman
+  prprocessor: true
```
Comment (ekohl) [inline config/repos.yaml]: You added this because it was already there in practice but not configured?

**Comment by ekohl** [comment]
There's also a typo in the commit message :)

File: config/repos.yaml (hunk 2 above)
Comment (ehelms) [reply config/repos.yaml]: I started adding it with the idea to track this but I should remove it.

File: scripts/update_webhook.rb (hunk 1 above)
Comment (ehelms) [reply scripts/update_webhook.rb]: Good question -- I don't actually know. But I can test it.

**Comment by ehelms** [comment]
I really wanted to just use the Ansible github_hooks module and make it real simple but its out of date and lacks secret support and specific event support.

**Comment by ehelms** [comment]
This is updated for HTTPS (I also updated and tested the repositories) as well as commit message updated.

**Pull request merged**